js/apps/admin-ui/src/client-scopes/details/MappingDetails.tsx

```diff
@@ -31,6 +31,8 @@ import { useFetch } from "../../utils/useFetch";
 import { useParams } from "../../utils/useParams";
 import { toClientScope } from "../routes/ClientScope";
 import { MapperParams, MapperRoute } from "../routes/Mapper";
+import { findProtocolMapperByClientScope } from "../../components/client-scope/protocolMapperGetter";
+import { findProtocolMapperByClient } from "../../components/client-scope/protocolMapperGetter";
```

Comment:
This import can be unified.
```suggestion
import { findProtocolMapperByClientScope, findProtocolMapperByClient } from "../../components/client-scope/protocolMapperGetter";
```

Reply:
@jonkoops Done, Thanks for the suggestion!